src/main/java/com/example/electronicstore/login/entity/User.java

```diff
@@ -0,0 +1,71 @@
+package com.example.electronicstore.login.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+	
+	private String firstName;
+	
+	private String lastName;
+	
+	private String password;
+	
+	private String email;
+
+	public Long getId() {
```

Comment:
you dont need getter/setter separately if you are adding @Data annotation. please refer this and update.
https://projectlombok.org/features/Data